pymeasure/instruments/siglenttechnologies/siglent_spdbase.py

```diff
@@ -138,137 +232,23 @@ def enable_local_interface(self, enable: bool = True):
             ``True``: enables the local interface
             ``False``: disables it.
         """
-        self.write(("*LOCK", "*UNLOCK")[enable])
+        self.write(("*UNLOCK" if enable else "*LOCK"))
```

Comment:
I think you do not need the second pair of parenthesis. Formerly it was necessary to create a tuple.

Reply:
Yep, onboard with that. I missed these